src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/MetricItemExtensions.cs

```diff
@@ -267,37 +268,12 @@ internal static OtlpMetrics.Metric ToOtlpMetric(this Metric metric)
                             }
                         }
 
-                        var exemplars = metricPoint.GetExemplars();
-                        foreach (var examplar in exemplars)
+                        if (metricPoint.TryGetExemplars(out var exemplars))
```

Comment:
I noticed we process exemplars only for Histograms here. Need to process for other MetricTypes as well. Can be a follow up as its not introduced in this PR.

Reply:
Ya I noticed that too. I had to fix it up in order to run stress tests for counters w/ exemplars. I have the code locally but thought best to do that as its own PR.